src/OrderLinesList/OrderLinesList.js

```diff
@@ -35,15 +36,16 @@ import { searchableIndexes } from './OrdersLinesSearchConfig';
 import OrderLinesListActionMenu from './OrderLinesListActionMenu';
 import LineExportSettingsModalContainer from './LineExportSettingModalContainer';
 
-const VENDOR_REF_NUMBER = 'vendorDetail.refNumber';
```

Comment:
is there a reason to delete constant?

Reply:
I'll return it back but doesn't make much sense now